docs/helper-environment-visitor.md

```diff
@@ -0,0 +1,285 @@
+---
+id: babel-helper-environment-visitor
+title: "@babel/helper-environment-visitor"
+sidebar_label: helper-environment-visitor
```

Comment:
Should we add it to the `sidebars.json`? I feel like we should have an "Advanced Topics" for such niche helpers.